posthog/urls.py

```diff
@@ -134,6 +134,49 @@ def authorize_and_redirect(request: HttpRequest) -> HttpResponse:
     )
 
 
+def link_and_redirect(request: HttpRequest) -> HttpResponse:
```

Comment:
Again this seems really similar to `authorize_and_redirect` above, why is it separate instead of using the same function? Or if they need to be different endpoints, then might be good to reuse the code..

Reply:
Yeah, this function is pretty similar. Will reuse the existing function.